client/rpcserver/handlers.go

```diff
@@ -913,7 +944,26 @@ Registration is complete after the fee transaction has been confirmed.`,
       "cancelling" (bool): Whether this order is in the process of cancelling.
       "canceled" (bool): Whether this order has been canceled.
       "tif" (string): "immediate" if this limit order will only match for one epoch.
-        "standing" if the order can continue matching until filled or cancelled.
+      "standing" if the order can continue matching until filled or cancelled.
```

Comment:
The extra indentation on `"standing"` should be restored since it's not a new field of the order, but part of the description for the `"tif"` field.

Reply:
opps, true that